src/analysis/Parse.jl

```diff
@@ -83,7 +83,7 @@ fix_linenumbernodes!(::Any, actual_filename) = nothing
 `expression_boundaries("sqrt(1)\n\n123") == [ncodeunits("sqrt(1)\n\n") + 1, ncodeunits("sqrt(1)\n\n123") + 1]`
 
 """
-function expression_boundaries(code::String, start=1)::Vector{<:Integer}
+function expression_boundaries(code::String, start=1)
```

Comment:
The return type is a form of documentation here

Reply:
Ah yes. I see:

https://github.com/fonsp/Pluto.jl/blob/de6a312b09d09081d549681d05d286033d894ce7/src/precompile.jl#L80-L81

Reply:
I meant more that this is a form of documentation for developers: I want to communicate that this function returns a `Vector{<:Integer}`, to make it more clear what the function does.

Just like how `find_first_vowel(::String)` would be ambiguous, but `find_first_vowel(::String)::Char` is clear.

Reply:
I think PrecompileSignatures works with the *input signatures* of functions, Rik will know more :)